src/index.html

```diff
@@ -1,13 +1,196 @@
+            <p class="section-1__card-text text animate-on-scroll-right">
+              Portable bluetooth
+            </p>
+            <p class="section-1__card-text animate-on-scroll-right">
+              $ 129.00
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section-2" id="wiew">
+        <h2 class="section-2__title title animate-on-scroll-right">
+          Browse Bose
+          <br>products by
+          category
+        </h2>
+        <div class="section-2__container">
+          <div class="section-2__img1 section-2__img--row-1"></div>
+
+          <div class="section-2__img2 section-2__img--row-1 animate-on-scroll-left"></div>
+
+          <p class="section-2__subtitle section-2__subtitle--row-2 animate-on-scroll-right">
+            Headphones & earbuds
+          </p>
+
+          <div class="section-2__img3 section-2__img--row-3 animate-on-scroll-right"></div>
+
+          <div class="section-2__img4 section-2__img--row-3 animate-on-scroll-right"></div>
+
+          <p class="section-2__subtitle section-2__subtitle--row-4 animate-on-scroll-right">
+            Speakers
+          </p>
+
+          <div class="section-2__img5 section-2__img--row-5 animate-on-scroll-left"></div>
+
+          <div class="section-2__img6 section-2__img--row-5 animate-on-scroll-left"></div>
+
+          <p class="section-2__subtitle section-2__subtitle--row-6 animate-on-scroll-right">
+            Audio sunglasses
+          </p>
+        </div>
+        <div class="section-2__button button animate-on-scroll-right">
+          <a href="#" class="button__title">View all</a>
+        </div>
+      </section>
+
+      <section class="section-3" id="about">
+        <div class="section-3__container">
+          <h2 class="section-3__title title animate-on-scroll-left">
+            Why buy direct
+            <br> from Bose
+          </h2>
+          <div class="section-3__text-box animate-on-scroll-left">
+            <p class="section-3__text text">Free 2-day shipping and returns</p>
+            <p class="section-3__text text">90-day risk-free trial</p>
+            <p class="section-3__text text">World class customer service</p>
+            <p class="section-3__text text">My Bose account management</p>
+            <p class="section-3__text text">A great product is more than what's
+              in the box. It's a promise of premium performance, world-class
+              support, and everything you expect from a trusted brand. It's just
+              one of many reasons why you'll shop with confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section-4" id="contact">
+        <div class="section-4__container">
+          <h2 class="section-4__title title animate-on-scroll-left">
+            Contact us
+          </h2>
+          <form action="#" method="get"
+            class="section-4__form animate-on-scroll-right">
```

Comment:
If we have 3 or more attributes, then all of them must be on a new line. Need to check and fix it everywhere